isis/src/control/apps/jigsaw/jigsaw.xml

```diff
@@ -1513,5 +1513,106 @@
+        <p>
+         This relative bundle turns on and parameterizes the camera twist and camera acceleration solve parameters. Turning on camera twist allows for the bundle to solve 
+         for the cameras rotation around the bore sight axis and does not require parameterization. Setting the camera solve parameter to acceleration is useful for 
+         line scan imagers and requires uncertainties to be set for the angles (deg), angular velocity (deg/s), and angular accelerations (deg/s**2). These values were 
+         set with increasing constraint because of the increasing affect alterations of higher derivative parameters have on the bundle solution.
+        </p>
+        <p>
+         The ‘overexisting’ flag allows the bundle solution to approximate the camera solution with a polynomial function as opposed to storing every individual measurement. 
+         This saves time and memory required for the bundle solution at the expense of accuracy. If one required the accuracy, overexisting could be used until the bundle 
+         is converging in a satisfactory way and then turned off for the final solution.
+        </p>
+        <p>
+         This bundle also lowers the max iterations to 10 (from default 50). Lowering the max iterations does not affect the bundle solution. However, setting a lower 
+         iteration limit can serve as a flag if you expect your network to bundle quickly. Finally, the sigma0 convergence criteria was not changed from its default 
+         value, it was only explicitly stated in this call.
+        </p>
+      </description>
+      <terminalInterface>
+        <commandLine>
+        jigsaw fromlist=cubes.lis cnet=relative_reg_jig4_edit.net onet=relative_reg_jig5.net update=no file_prefix=jig5 sigma0=1.0e-10 maxits=10 \
+               twist=yes camsolve=accelerations overexisting=yes camera_angles_sigma=.25 camera_angular_velocity_sigma=.1 camera_angular_acceleration_sigma=.01
+        </commandLine>
+        <description>
+        </description>
+      </terminalInterface>
+    </example>
+
+    <example>
+      <brief>Run of jigsaw parameterized for Kaguya Terrain Camera (TC) images and <b>an intermediate ground control</b> network covering the Apollo 15 landing site. </brief>
+      <description>
+        <p>
+         This example jigsaw bundle is run directly after adding ground control points to the previous relative network. With ground control points inserted into the 
+         bundle solution, we can expand the bundle solve parameters to attempt solving for point radius values and the position of the spacecraft.
+        </p>
+        <p>
+         Ground points are weighted more heavily in the bundle than relative control points and adding parameters adds more complexity to the bundle solution. Therefore, it 
+         is common to create and smith a network with only relative points and add ground point in AFTER the relative network (and its bundle solution) is of sufficient quality. 
```

Comment:
I would avoid the word `smith` as it could be confused with `smithed` kernels that are produced from bundle adjusted images. I would instead use something like `refined` or `improve`.